src/esp/gfx/PbrShader.cpp

```diff
@@ -129,6 +170,11 @@ PbrShader::PbrShader(Flags originalFlags, unsigned int lightCount)
                      ? rs.getString("shadowsVSM.glsl") + "\n"
                      : "")
       .addSource(rs.getString("pbrCommon.glsl") + "\n")
```

Comment:
The `\n` shouldn't be necessary for the files anywhere, the files themselves should have a trailing newline. Git itself warns if they don't and I think clang-format even enforces that.

Also, the append of a `"\n"` will cause an unnecessary string allocation & copy, while without it'd be a zero-copy operation as the source string view is constant global memory.